recipes/mercat/meta.yaml

```diff
@@ -0,0 +1,53 @@
+
+package:
+  name: mercat
+  version: "0.1"
+
+source:
+   git_url: https://github.com/pnnl/mercat.git
+#  git_rev: 0.1
+
+# build:
+#  number: 0.1
+
+build:
+  script: python setup.py install
+
+
+preserve_egg_dir: yes
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scikit-bio
+    - prodigal
+    - trimmomatic
+    - dask
+    - setuptools
+    - pandas
+    - numpy
+    - humanize
+    - plotly
+    - psutil
+    - joblib
+
+
+#test:
```

Comment:
A simple test passes on MacOSX build, but not for Linux. When trying to import python modules, I get the error  
> ImportError: libstdc++.so.6: cannot open shared object file
I tried specifying libgcc, gcc as runtime dependencies in the recipe, but I still see it.
[https://travis-ci.org/ajaypanyala/bioconda-recipes/jobs/203529745](https://travis-ci.org/ajaypanyala/bioconda-recipes/jobs/203529745)